src/prompt.ts

```diff
@@ -0,0 +1,84 @@
+import * as inquirer from "inquirer";
+import * as _ from "lodash";
+
+import * as FirebaseError from "./error";
+
+export interface Choice {
```

Comment:
Since this is an exported type, I'd add a jsdoc here to add an example for how this is used to add more context.

Reply:
It's been removed.